jpo-geojsonconverter/src/main/resources/application.yaml

```diff
@@ -44,6 +46,9 @@ kafka.topics:
     - name: topic.ProcessedRtcm
       cleanupPolicy: delete
       retentionMs: 300000
+    - name: topic.ProcessedTim
```

Comment:
Do we have a jpo-utils commit that includes this topic in the Kafka topic creation script?

Reply:
I think we still need to have a jpo-utils commit reference to have the added topic. I know the GJC makes the topics if they don't exist but we have every other topic in there so might as well be consistent.